### ISSUE TITLE
nemo broken

### ISSUE THREAD

**Issue by jeffhammond** [m-a-d-n-e-s-s/madness#173]
From https://travis-ci.org/m-a-d-n-e-s-s/madness/jobs/99759048:
```
Linking CXX executable nemo
Undefined symbols for architecture x86_64:
  "madness::info::cheminfo_git_commit()", referenced from:
      _main in nemo.cc.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [src/examples/nemo] Error 1
make[1]: *** [src/examples/CMakeFiles/nemo.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

**Comment by jeffhammond** [comment]
@justusc I see this happens on Mac+Clang but not Linux+Clang.  Is CMake building this program while Autotools is not?  We should reconcile the two buildsystems, preferably such that they both work.  I think the issue in `nemo.cc` is trivial, but I haven't had time to debug it.

**Comment by justusc** [comment]
The CMake build did not include a need source file in the MADchem lib. This issue is resolved in the Travis branch.

**Comment by jeffhammond** [comment]
I assume @justusc will close this since he fixed it.

**Comment by justusc** [comment]
Fixed via 8463513002b3696de0ac12cf9581c2398476871d.
